Policies/Kubernetes/enforce-apparmor-profile/examples-good/example_good.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
```

Comment:
In examples-good folder, can you add an example file for assignment parameters? Create a file by name azurepolicy.assignment.parameters.json and provide content as below...

{
  "parameters": {
    "allowedServicePortsList": { "value": [ "8080" ] },
    "excludedNamespaces": { "value": [ "kube-system", "azure-arc" ] }
  }
}

Reply:
I've added this